seatunnel-dist/release-docs/LICENSE

```diff
@@ -329,7 +329,7 @@ The text of each license is the standard Apache 2.0 license.
      (Apache License, Version 2.0) Apache Hadoop YARN Client (org.apache.hadoop:hadoop-yarn-client:3.0.0 - no url defined)
      (Apache License, Version 2.0) Apache Hadoop YARN Common (org.apache.hadoop:hadoop-yarn-common:3.0.0 - no url defined)
      (Apache License, Version 2.0) Apache HttpAsyncClient (org.apache.httpcomponents:httpasyncclient:4.1.2 - http://hc.apache.org/httpcomponents-asyncclient)
-     (Apache License, Version 2.0) Apache HttpClient (org.apache.httpcomponents:httpclient:4.5.2 - http://hc.apache.org/httpcomponents-client)
+     (Apache License, Version 2.0) Apache HttpClient (org.apache.httpcomponents:httpclient:4.5.13 - http://hc.apache.org/httpcomponents-client)
      (Apache License, Version 2.0) Apache HttpClient Mime (org.apache.httpcomponents:httpmime:4.5.2 - http://hc.apache.org/httpcomponents-client)
```

Comment:
you need to add `commons-codec` here,

I checked NOTICE and he didn't change, so just make the changes here.

Reply:
This process is tedious, can we simplify it？

Reply:
License is very important, it is required. I don't currently see a better approach (automation) for other projects.

Reply:
Before add common-codec, the license checker is successful. now, it is failed.

Reply:
That's not your problem, hi, @asdf2014 can you take a look at this?

Reply:
> That's not your problem, hi, @asdf2014 can you take a look at this?

Thanks.

Reply:
@zhaomin1423 Have you looked the log

Reply:
@CalvinKirs It is normal, we should be glad that the checker is working

Reply:
Am I missing something? I saw that the license item has been done, and this error is very strange.

Reply:
@CalvinKirs see my last comment